flows/load_image.py

```diff
@@ -84,6 +85,9 @@ def load_image(FILENAME):
 		image.image = np.asarray(hdul[0].data, dtype='float64')
 		image.shape = image.image.shape
 
+		image.head = hdr
+		image.exthdu = [hdu.copy() for hdu in hdul[1:]]
```

Comment:
Is this used anywhere? It is an expensive operation if not used for anything and we might as well just use the `HDUList` from the original FITS file if it is needed.

Reply:
removed